src/Core/System/Resources/config/basicInformation.xml

```diff
@@ -150,9 +150,9 @@
             <name>useDefaultCookieConsent</name>
             <bordered>bordered</bordered>
             <label>Use Default Cookie Notification</label>
-            <label lang="de-DE">Standard Cookiehinweis verwenden</label>
+            <label lang="de-DE">Standard-Cookiehinweis verwenden</label>
```

Comment:
```suggestion
            <label lang="de-DE">Standard-Cookie-Hinweis verwenden</label>
```

Reply:
I thought of it, but it looks weirder to me 🙈